### PR TITLE
Adding MAM Targeting

### PR DESCRIPTION
**Pull request by SylLutze**

### CODE REVIEW

File: docs/source/models/intervention_models/wasting_treatment/compas_trial_wasting_treatment/index.rst [hunk 1]
```diff
@@ -244,6 +244,31 @@ The coverage propensity for wasting treatment parameter :math:`C` for any given
 
   This strategy assumes that simulants who are treated for MAM and SAM once are no more likely to be treated again than simulants who have never been treated for SAM or MAM (despite need).
 
+Targeted Coverage for MAM Treatment
++++++++++++++++++++++++++++++++++++
+
+In accordance with `new guidelines from the WHO <https://files.magicapp.org/guideline/a3fe934f-6516-460d-902f-e1c7bbcec034/published_guideline_7330-1_1.pdf>`_ on the prevention and management of acute 
+malnutrition, we are including an option for targeted MAM treatment. The guidelines for 
+receiving MAM treatment include numerous factors for children to be consider for MAM 
+treatment including:
```
Comment (alibow) [inline docs/source/models/intervention_models/wasting_treatment/compas_trial_wasting_treatment/index.rst:253]:
```suggestion
treatment including the presence of *any* of the following criteria: 
```

**Review by alibow — APPROVED** [review]
Looks great, thanks Syl! Maybe we can also make a quick note about how we will plan to address the third criterion at some point too.

**Comment by alibow** [comment]
Oh and also maybe a note that we'd like this to be able to be turned on and off? Like for a scenario with targeted MAM versus a scenario with universal MAM. And then maybe we can add a run request to the child model request table for a targeted MAM scenario with maybe some extra stratifications so we can do V&V

**Review by hussain-jafari — APPROVED** [review]

File: docs/source/models/intervention_models/wasting_treatment/compas_trial_wasting_treatment/index.rst [hunk 2]
```diff
@@ -244,6 +244,31 @@ The coverage propensity for wasting treatment parameter :math:`C` for any given
 
   This strategy assumes that simulants who are treated for MAM and SAM once are no more likely to be treated again than simulants who have never been treated for SAM or MAM (despite need).
 
+Targeted Coverage for MAM Treatment
++++++++++++++++++++++++++++++++++++
+
+In accordance with `new guidelines from the WHO <https://files.magicapp.org/guideline/a3fe934f-6516-460d-902f-e1c7bbcec034/published_guideline_7330-1_1.pdf>`_ on the prevention and management of acute
```
Comment (aflaxman) [inline docs/source/models/intervention_models/wasting_treatment/compas_trial_wasting_treatment/index.rst:250]: is magicapp.org really the definitive domain to get these guidelines?  that is so spammy!

**Review by aflaxman — APPROVED** [review]

**Review by patricktnast — APPROVED** [review]

File: docs/source/models/intervention_models/wasting_treatment/compas_trial_wasting_treatment/index.rst (hunk 2 above)
Comment (SylLutze) [reply docs/source/models/intervention_models/wasting_treatment/compas_trial_wasting_treatment/index.rst:250]: Okay I was curious and it appears yes. In other press releases they link to magicapp as well so I think this is the "best" link.

**Comment by SylLutze** [comment]
> Oh and also maybe a note that we'd like this to be able to be turned on and off? Like for a scenario with targeted MAM versus a scenario with universal MAM. And then maybe we can add a run request to the child model request table for a targeted MAM scenario with maybe some extra stratifications so we can do V&V

Hey @alibow might be good to have you take a second look at this since the changes were not insignificant. There are already scenarios included for targeted vs universal MAM in the concept model doc - so no need to add those. I did add in model run requests. We hadn't decided on a model numbering system for wave 2 so I decided to make new tables for wave 2 and start with model 9 but happy to change that.

**Comment by alibow** [comment]
Yeah I think your changes look great, @SylLutze! I think breaking wave II into a separate table makes sense also, but maybe we can see if @hussain-jafari or @rmudambi have other preferences before merging?

**Comment by hussain-jafari** [comment]
Works for me!

**Pull request merged**